AstSemantics.md

```diff
@@ -531,3 +531,12 @@ round-to-nearest ties-to-even rounding.
 Truncation from floating point to integer where IEEE 754-2008 would specify an
 invalid operation exception (e.g. when the floating point value is NaN or
 outside the range which rounds to an integer in range) traps.
+
+## Feature test
+
+To support [feature testing](FeatureTest.md), an AST node would be provided:
+
+  * `has_feature`: return whether the given feature is supported, identified by string
```

Comment:
Agreed that once we have something bigger than an instance, things get fuzzy.  However, it does seem valuable to know that if module A probes a feature and chooses to load module B, B will get the same result.  I think we could bound this by saying "within the same user agent session" which is admittedly fuzzy, but probably good enough for now.